src/org/labkey/test/components/react/dialogs/BulkCreateSamplesDialog.java

```diff
@@ -0,0 +1,160 @@
+package org.labkey.test.components.react.dialogs;
```

Comment:
This is a LabKey developed component
```suggestion
org.labkey.test.components.labkey.ui.samples;
```

Reply:
done